simple/stats/data.py

```diff
@@ -34,6 +34,7 @@
 _PREDICATE_POPULATION_TYPE = "populationType"
 _PREDICATE_MEASURED_PROPERTY = "measuredProperty"
 _PREDICATE_STAT_TYPE = "statType"
+_PREDICATE_UTTERANCE_TEMPLATE = "utteranceTemplate"
```

Comment:
@pradh this schema is very confusing, why would this be the nl sentence? From previous discussion with Keyur, feels like this is a transformation function rather than schema or the actual description.

Reply:
That's right, this can originally be "population of $Place" too.   Would we prefer to use something like "searchDescription" ?

Reply:
yea, searchDescription sg

Reply:
Updated to use `searchDescription`.